dracut/30ignition/ignition-diskful-ready.target

```diff
@@ -0,0 +1,14 @@
+# This target contains Ignition units that should only run when we have a
```

Comment:
I don't see any code installing this target?

Reply:
Or maybe it was meant to be a separate commit?  This commit message just talks about `boot.mount`.

Also the commit message subject is a bit redundant, how about:

> boot.mount: New unit to centralize handling of /boot

Reply:
Lovely cruft. Will be dropped in the next push.

Reply:
FWIW, `ignition-diskful-ready.target` was something I was playing with as a means to work around the races for disk-labels. And then I realized it was not needed and introduced more complexity than was needed. That and it likely won't fix the problem.